message-index/site.hs

```diff
@@ -151,6 +152,10 @@ main = hakyll $ do
         >>= loadAndApplyTemplate "templates/default.html" indexCtx
         >>= relativizeUrls
 
+  -- Needed for flagInfo below
+  match "warning-sets/warning-sets-9.5.txt" $ do
```

Comment:
I'd like to get to a point where site.hs doesn't need rebuilding, and only the generated site itself needs (re)building, and this seems like something we'd need to change with some frequency. Can we wildcard it?

Reply:
Is adding a line to and rebuilding `site.hs` worse than adding more pages? They seem similarly difficult from my perspective. In each case, the workflow is "test local, merge, push, wait for deploy to happen".

Reply:
My thought process is that we could distribute the binary as part of the repository, so people don't have to install ghc+hakyll+dependencies to contribute, if that makes sense.

Reply:
> I'd like to get to a point where site.hs doesn't need rebuilding, and only the generated site itself needs (re)building, and this seems like something we'd need to change with some frequency. Can we wildcard it?

No more often than adding new compilers.

Once we want to document multiple versions we can make it so that `site.hs` doesn’t have to change and will pick up all `warning-sets/warning-sets*.txt` files and they will all automatically show up. But while we have only one version, let’s keep it simple, and worry about that then.

Reply:
Fair.

Reply:
I agree with this approach.